libs/platform/user/nmr_impl.cpp

```diff
@@ -301,7 +301,7 @@ _nmr::remove(_Inout_ collection_t& collection, _In_ collection_t::value_type::fi
 
     // Wait for bindings to reach zero if requested.
     if (it->second.binding_count > 0) {
-        for (;;) {
+        for (size_t i = 0; i < 12; i++) {
```

Comment:
`12` seems a curious retry value.  A comment would be helpful. 

A nit: use `constexpr size_t max_retries = 12` and use that instead of a magic number?